### PR TITLE
Add an example to draw a radius with Turf.js

### PR DESCRIPTION
**Pull request by clementgayvallet**
This PR adds an example to show how to draw a radius with Turf.js. This is a typical use-case when a user wants to display an approximation of a location such as a real estate property.

The access token used in this example is dedicated and will work from `localhost` and `maplibre.org` origins.

## Launch Checklist

 - [x] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [x] Briefly describe the changes in this PR.

### CODE REVIEW

File: test/examples/draw-a-radius.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        body { margin: 0; padding: 0; }
+        html, body, #map { height: 100%; }
+    </style>
+</head>
+<body>
+<script src="https://cdn.jsdelivr.net/npm/@turf/turf@7/turf.min.js"></script>
+<div id="map"></div>
+<script>
+    // Don't forget to replace with your own access token
+    const accessToken = '0SMf4Ib7tchV9KXSTr9hHKZhtsPaf4XP2FW4gs4UO7E97cPae1CEuTf7owi0NLdn';
```
Comment (HarelM) [inline test/examples/draw-a-radius.html:20]: Please use `demotiles` instead.

File: test/examples/draw-a-radius.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            steps: 64,
+            units: 'kilometers'
+        };
+        const circle = turf.circle(radiusCenter, radius, options);
+
+        // Add a fill layer with some transparency
+        map.addLayer({
+            id: 'location-radius',
+            type: 'fill',
+            source: {
```
Comment (HarelM) [inline test/examples/draw-a-radius.html:43]: I would consider adding a source and use that source in both layers, I believe it is more readable.

**Comment by HarelM** [comment]
Thanks for taking the time to open this PR!
This is a good example.

File: test/examples/draw-a-radius.html (hunk 1 above)
Comment (clementgayvallet) [reply test/examples/draw-a-radius.html:20]: I first thought about using `demotiles`, but the style doesn't have street-level details, so the example would make less sense with a country-wide radius. Also, I was inspired by this PR (#3592) where a style with an API key is used.

Comment (HarelM) [reply test/examples/draw-a-radius.html:20]: Yes, I missed that, it's less than ideal, can't you simply use OSM raster baselayer?

**Review by HarelM — APPROVED** [review]
THANKS!

**Pull request merged**